### PR TITLE
#500: [Babel] Update Babel (closes #500)

### LINKED ISSUE #500: [Babel] Update Babel
## requirements
- update `babel-*` to `^v6`
- define a `.babelrc`

## specs
- be sure all app **tasks** work, i.e. `npm run build, npm run test, ...`

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by ascariandrea**
Issue #500

## Tests perfomed
- `npm start` ✔️ 
- `npm run build` ✔️ 
- `npm run test` ✔️

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,6 +88,7 @@
     "require-noop": "0.0.2",
     "sass-loader": "^3.1.2",
     "style-loader": "^0.13.0",
+    "walk": "^2.3.9",
```
Comment (FrancescoCioria) [inline package.json]: ?

File: package.json [hunk 2]
```diff
@@ -34,6 +35,7 @@
     "bowser": "^1.0.0",
     "classnames": "^2.2.3",
     "css-element-queries": "^0.3.2",
+    "debug": "^2.2.0",
```
Comment (FrancescoCioria) [inline package.json]: are we using this?

Comment (ascariandrea) [inline package.json]: Give me a sec 😝

**Comment by FrancescoCioria** [comment]
👍

**Comment by gabro** [comment]
FYI, this broke LOL (and possibly other related projects).

https://drone-omnilab.our.buildo.io/buildo/aliniq/7495

@FrancescoCioria I don't remember the strategy we decided for fixing a version of BRC on the projects, did it involve LOL too?

**Pull request merged**